### DISCUSSION TITLE
Is that normal ?

### DISCUSSION THREAD

**Discussion by lynn2910** [community/community#7545] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/73949405/141136158-fa9c6cb2-6fa2-4da8-89c3-f61a3fbc7ece.png)

In this screen, I use the multiple line write fonctionnality from Visual Studio Code, but GitHub Copilote give me only one line

I don't know if this is normal, but it's maybe a bug or a good fonctionnality that can be add !

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey, 

Yes, we currently don't support multiple cursors in Copilot.
